src/pages/Login.jsx

```diff
@@ -13,29 +12,27 @@ function Login() {
     handleSubmit,
     formState: { errors }
   } = useForm()
-  const { setAuth } = useAuth()
+  const { userLogin } = useAuth()
   const navigate = useNavigate()
   const [responseError, setResponseError] = useState()
 
   const onSubmit = async (data) => {
-    const response = await login(data)
+    const response = await userLogin(data)
     if (!response.success) {
       setResponseError(response.error)
       return
```

Comment:
🍩  Esto podrias dejarlo en una sola linea
```suggestion
      return setResponseError(response.error)
```